### PR TITLE
[tutorials] Update intro tutorial step 7

### LINKED ISSUE #691: [tutorials] Missing css import
On Step 7/8 of the tutorial the provided code is missing the `css` import. While this is pretty easy to figure out, it may cause some to stumble as there is no reference in the tutorial that it needs to be imported. Without it, the preview area just renders blank.

### PR DESCRIPTION
**Pull request by augustjk**
Fixes #691 

- Add js file to before to preserve css import
- Update after file to remove property only used in next step

### CODE REVIEW

File: packages/lit-dev-content/samples/tutorials/intro-to-lit/06/after/todo-list.ts [hunk 1]
```diff
@@ -15,13 +15,11 @@ export class ToDoList extends LitElement {
     }
   `;
 
-  @property()
+  @property({attribute: false})
```
Comment (AndrewJakubowicz) [inline packages/lit-dev-content/samples/tutorials/intro-to-lit/06/after/todo-list.ts:18]: Great catch. This made me realize there are a couple other inconsistencies between Step 6 and Step 7.

In Step 6 there's indentation that changes on the solve.
Step 6 uses `class` whilst Step 7 uses `export class`.

However they aren't blocking. Maybe we track in an issue and resolve separately.

File: packages/lit-dev-content/samples/tutorials/intro-to-lit/06/after/todo-list.ts [hunk 2]
```diff
@@ -15,13 +15,11 @@ export class ToDoList extends LitElement {
...
   listItems = [
     { text: 'Make to-do list', completed: true },
     { text: 'Add some styles', completed: true }
   ];
-  @property()
```
Comment (AndrewJakubowicz) [inline packages/lit-dev-content/samples/tutorials/intro-to-lit/06/after/todo-list.ts:23]: Great fix!

**Review by AndrewJakubowicz — APPROVED** [review]
Nice. I also manually did the tutorial steps modified and everything works great when following the tutorial text.

This is a strict improvement since it fixes the JS tutorial. I don't think my nits block this change.

**Review by e111077 — CHANGES_REQUESTED** [review]
Please add an exemption here for that TS file

https://github.com/lit/lit.dev/blob/324ad70458663f8f7c4960a3f60cbe8d587d2a83/packages/lit-dev-content/samples/tsconfig.json#L18

Also that highlighted one, do you mind changing that to `tutorials/intro-to-lit/01/before/my-element.ts` while you're there?

**Comment by e111077** [comment]
Hmm nevermind it seems like Al says it'll automatically know. Do you mind removing that line in the tsconfig then?

**Comment by augustjk** [comment]
> Hmm nevermind it seems like Al says it'll automatically know. Do you mind removing that line in the tsconfig then?

Good find! It has been removed.

File: packages/lit-dev-content/samples/tutorials/intro-to-lit/06/after/todo-list.ts (hunk 1 above)
Comment (arthurevans) [reply packages/lit-dev-content/samples/tutorials/intro-to-lit/06/after/todo-list.ts:18]: Mea culpa here. It's really hard to maintain all the before/after versions (@e111077's recent changes should help a ton with that). Currently the `{attribute: false}` is also missing in the previous step where this property is added.

https://github.com/lit/lit.dev/blob/main/packages/lit-dev-content/samples/tutorials/intro-to-lit/05/before/todo-list.ts#L6

Looks like the everything _before_ step 7 omits the export statement.

**Review by e111077 — APPROVED** [review]

**Pull request merged**